lib/server-starter.js

```diff
@@ -90,7 +92,23 @@ class Server extends EventEmitter {
     });
 
     // Should be resolved by the "spawn" event, but that is only supported in Node 15+
-    return Promise.resolve;
+    if (process.platform !== 'win32') return Promise.resolve;
+    return new Promise((resolve) => {
+      const retryTime = 60;
+      const now = new Date();
+      const timeToStop = new Date(now.getTime() + win32Timeout);
+      const port = this.port;
+      (function loop () {
+        const connection = net.connect(port, resolve);
+        connection.on('error', err => {
+          if (err.code === 'ECONNREFUSED') {
+            if (new Date() < timeToStop) {
```

Comment:
And if it times out we just do nothing?

Reply:
yes the caller will take care, I mean will recive the exact same error

Reply:
How does the error reach the caller?

Reply:
The logic is like:

- wait up to win32Timeout mS, as long as the error you are getting is ECONNREFUSED.
- if you could connect, resolve the promise
- if you get other error, you will not retry

so Ooops, I need to resolve at the end of the anonymous function (or reject, but that will be handled much better by the caller, when it connect)